main/modes/jukebox/jukebox.c

```diff
@@ -150,694 +145,490 @@ static const char str_play[]       = ": Play";
-    jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = true;
-    songIdx++;
-
-    if (songIdx != jukebox->musicCategories[catIdx].numSongs)
-    {
-        ESP_LOGE(JK_TAG, "BGM %d: %d != %d\n", catIdx, songIdx, jukebox->musicCategories[catIdx].numSongs);
-    }
-    catIdx++;
-
-    // Jukebox category
-    songIdx                                       = 0;
-    jukebox->musicCategories[catIdx].categoryName = jukeboxMode.modeName;
-    jukebox->musicCategories[catIdx].numSongs     = 1;
-    jukebox->musicCategories[catIdx].songs = calloc(jukebox->musicCategories[catIdx].numSongs, sizeof(jukeboxSong));
-
-    // Jukebox songs
-
-    // TODO: get these midis from Dac
-    // jukebox->musicCategories[catIdx].songs[songIdx].name = "Hot Rod";
-    // loadSong("hotrod.sng", &jukebox->musicCategories[catIdx].songs[songIdx].song, true);
-    // jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = true;
-    // songIdx++;
-
-    jukebox->musicCategories[catIdx].songs[songIdx].name = "Fauxrio Kart";
-    loadSong("Fauxrio_Kart.sng", &jukebox->musicCategories[catIdx].songs[songIdx].song, true);
-    jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = true;
-    songIdx++;
-
-    // jukebox->musicCategories[catIdx].songs[songIdx].name = "The Lake";
-    // loadSong("thelake.sng", &jukebox->musicCategories[catIdx].songs[songIdx].song, true);
-    // jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = true;
-    // songIdx++;
-
-    // jukebox->musicCategories[catIdx].songs[songIdx].name = "Ya like jazz?";
-    // loadSong("yalikejazz.sng", &jukebox->musicCategories[catIdx].songs[songIdx].song, true);
-    // jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = true;
-    // songIdx++;
-
-    // jukebox->musicCategories[catIdx].songs[songIdx].name = "Banana";
-    // loadSong("banana.sng", &jukebox->musicCategories[catIdx].songs[songIdx].song, true);
-    // jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = true;
-    // songIdx++;
-
-    if (songIdx != jukebox->musicCategories[catIdx].numSongs)
+jukeboxSong_t music_galacticBrickdown[] = {
     {
-        ESP_LOGE(JK_TAG, "BGM %d: %d != %d\n", catIdx, songIdx, jukebox->musicCategories[catIdx].numSongs);
-    }
-    catIdx++;
-
-    // Credits category
-    songIdx                                       = 0;
-    jukebox->musicCategories[catIdx].categoryName = "Credits";
-    jukebox->musicCategories[catIdx].numSongs     = 1;
-    jukebox->musicCategories[catIdx].songs = calloc(jukebox->musicCategories[catIdx].numSongs, sizeof(jukeboxSong));
-
-    // Credits songs
-
-    jukebox->musicCategories[catIdx].songs[songIdx].name = "Credits";
-    loadSong("credits.sng", &jukebox->musicCategories[catIdx].songs[songIdx].song, true);
-    jukebox->musicCategories[catIdx].songs[songIdx].song.shouldLoop = false;
-    songIdx++;
+        .filename = "stereo.sng",
+        .name     = "DELETE ME",
```

Comment:
Um, what's this?

Reply:
Needed something in there or it would crash 😎